sdk/python/kfp/dsl/importer_node.py

```diff
@@ -25,23 +25,28 @@
 from kfp.dsl.types import type_utils
 
 URI_KEY = 'uri'
-OUTPUT_KEY = 'artifact'
 METADATA_KEY = 'metadata'
+DEFAULT_OUTPUT_KEY = 'artifact'
+DEFAULT_COMPONENT_NAME = 'importer'
```

Comment:
I believe `.set_display_name(...)` already serves this purpose:

```python
@dsl.pipeline
def my_pipeline():
    dsl.importer(
        artifact_uri='foo',
        artifact_class=dsl.Artifact).set_display_name('my component')
```
Can we omit the `component_name` parameter from this PR?